### PR TITLE
docs: explain how to circleci

### PR DESCRIPTION
**Pull request by philsturgeon**
Added some documentation explaining CI in general, as we had some out of date stuff saying "JUnit was coming soon" and its been out for ages.

### CODE REVIEW

**Review by mnaumanali94 — APPROVED** [review]

File: docs/guides/8-continuous-integration.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+workflows:
+  sample:
+    jobs:
+      - lint
+```
+
+Change the `openapi.yaml` to point to whatever documents you want to lint, and use -f (format) to pick the JUnit output format. This is a standard test format that many CI servers understand, and means you should be able to see the errors in the Test interface.
+
+![On the CircleCI build results page there is a tab called Tests, which will show Spectral results so long as the junit format has been enabled]()
```
Comment (P0lip) [inline docs/guides/8-continuous-integration.md:43]: This feels weird.
I suppose you wanted to link to a URL?

File: docs/guides/8-continuous-integration.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```yaml
+version: 2.1
+
+orbs:
+  node: circleci/node@4.1
+
+jobs:
+  lint:
+    docker:
+      - image: cimg/node:15.1
```
Comment (P0lip) [inline docs/guides/8-continuous-integration.md:20]: We should probably refer to the latest LTS version (currently it's 14).

File: docs/guides/8-continuous-integration.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    steps:
+      # Checkout the code as the first step.
+      - checkout
+      # Create a folder for results to live in
+      - run: "[ -d lint-results ] || mkdir lint-results"
+      - run:
+          name: Run Spectral Lint
+          command:
+            npx @stoplight/spectral lint openapi.yaml
```
Comment (P0lip) [inline docs/guides/8-continuous-integration.md:29]: Should we indicate that more than multiple files or glob patterns can be provided?

**Review by P0lip — COMMENTED** [review]
Looks cool!

File: docs/guides/8-continuous-integration.md (hunk 3 above)
Comment (philsturgeon) [reply docs/guides/8-continuous-integration.md:29]: My theory was that we arent teaching the CLI here, we are teaching how to CI.

File: docs/guides/8-continuous-integration.md (hunk 1 above)
Comment (philsturgeon) [reply docs/guides/8-continuous-integration.md:43]: Fixed, this is an image. I added the alt then forgot the src. :D

File: docs/guides/8-continuous-integration.md (hunk 3 above)
Comment (mnaumanali94) [reply docs/guides/8-continuous-integration.md:29]: Agree with Phil. Lets keep it simple here.

File: docs/guides/8-continuous-integration.md (hunk 2 above)
Comment (mnaumanali94) [inline docs/guides/8-continuous-integration.md:20]:
```suggestion
      - image: cimg/node:14.17.0
```

**Review by mnaumanali94 — APPROVED** [review]

**Pull request merged**